internal/uidriver/glfw/run_notsinglethread.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !ebitensinglethread
+// +build darwin freebsd linux windows
+// +build !android
+// +build !ios
+
+package glfw
+
+import (
+	"github.com/hajimehoshi/ebiten/v2/internal/driver"
+	"github.com/hajimehoshi/ebiten/v2/internal/graphicscommand"
+	"github.com/hajimehoshi/ebiten/v2/internal/thread"
+)
+
+func (u *UserInterface) Run(uicontext driver.UIContext) error {
+	u.context = uicontext
+
+	// Initialize the main thread first so the thread is available at u.run (#809).
+	u.t = thread.NewOSThread()
```

Comment:
`thread.New` is also called in internal/uidriver/mobile so please fix that. Note that mobiles cannot enable the single-thread mode since a rendering thread is given externally. Then, just renaming it is fine. Don't forget to rename `*thread.Thread` to `*thread.OSThread` (or `thread.Thread`).

Reply:
I think I did this already or I am misunderstanding.

Reply:
You are right. Thanks!